packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -55,6 +55,7 @@ export interface IDetailsListState {
   isSizing?: boolean;
   isDropping?: boolean;
   isSomeGroupExpanded?: boolean;
```

Comment:
Can't the parent component just use `componentRef` and `scrollIndexIntoView` on the `IDetailsList` instance?